index.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable */
+const BASE_URL = 'https://trektravel.herokuapp.com/trips';
+const continents = [];
+const getTrips = (url) => {
+  $.get(url, (response) => {
+    console.log('success!');
+    console.log(response);
+    response.forEach((trip) => {
+      const tripInfo = `<li data-id= ${trip.id} id = "trip-${trip.id}"><strong>${trip.name}</strong> - ${trip.weeks} week(s) in ${trip.continent} </li>`;
+      $('#trip-list ul').append(tripInfo);
+      if (!continents.includes(trip.continent.toLowerCase()) && trip.continent) {
```

Comment:
if `trip.continent` is null, the code breaks. Surprisingly enough, if you change the order of this conditional, then it'll work:
```javascript
if (trip.continent && !continents.includes(trip.continent.toLowerCase())) {
```

In a conditional with an `&&`, if `trip.continent` is falsey, it won't bother evaluating the rest of the conditional